odoo/http.py

```diff
@@ -703,6 +704,8 @@ def serialize_exception(e):
         tmp["exception_type"] = "access_denied"
     elif isinstance(e, odoo.exceptions.ValidationError):
         tmp["exception_type"] = "validation_error"
+    elif isinstance(e, odoo.exceptions.ViewError):
+        tmp["exception_type"] = "view_error"
```

Comment:
Ouch!  Do you really need a new category of error here?

Reply:
The web client depends on this to name the modal among other things, otherwise it's treated as a generic server error... It's not strictly necessary but I don't see why it's so bad

Reply:
Avoid adding errors